packages/compass-assistant/src/assistant-chat.spec.tsx

```diff
@@ -1,12 +1,24 @@
 import React from 'react';
-import { render, screen, userEvent } from '@mongodb-js/testing-library-compass';
+import {
+  render,
+  screen,
+  userEvent,
+  waitFor,
+} from '@mongodb-js/testing-library-compass';
 import { AssistantChat } from './assistant-chat';
 import { expect } from 'chai';
 import { createMockChat } from '../test/utils';
 import type { AssistantMessage } from './compass-assistant-provider';
 
-// TODO: some internal logic in lg-chat breaks all these tests, re-enable the tests
-describe.skip('AssistantChat', function () {
+describe('AssistantChat', function () {
+  // Mock scrollTo method for DOM elements to prevent test failures
+  before(function () {
+    if (!Element.prototype.scrollTo) {
+      Element.prototype.scrollTo = function () {
+        // No-op implementation for testing
+      };
+    }
+  });
```

Comment:
We are collecting all those under https://github.com/mongodb-js/compass/blob/main/configs/mocha-config-compass/register/jsdom-extra-mocks-register.js, do you mind moving this also?